src/lib.rs

```diff
@@ -7,6 +7,8 @@ pub mod user;
 /// All messages broadcasted by Mostro daemon are Parameterized Replaceable Events
 /// and use 30078 as event kind
 pub const NOSTR_REPLACEABLE_EVENT_KIND: u64 = 30078;
+pub const PROTOCOL_MAJOR_VER: u8 = 1;
+pub const PROTOCOL_MINOR_VER: u8 = 0;
```

Comment:
can you elaborate why you think is a good idea to have messages with major and minor versions?